aspnetcore/security/anti-request-forgery.md

```diff
@@ -159,7 +159,7 @@ Calling <xref:Microsoft.Extensions.DependencyInjection.MvcServiceCollectionExten
 
 ## Multiple browser tabs and the Synchronizer Token Pattern
 
-With the Synchronizer Token Pattern, only the most recently loaded page contains a valid antiforgery token. Using multiple tabs can be problematic. For example, if a user opens multiple tabs:
+With the Synchronizer Token Pattern, only the most recently loaded page is guaranteed to contain a valid antiforgery token. Apps that wish to support multiple tabs should test supported browsers and log failures. ***Using multiple tabs can be problematic***. For example, if a user opens multiple tabs:
```

Comment:
The intro to the bulleted list is now out of sync with the list itself:

* is guaranteed to contain
* contains

Consider changing the second bullet with something similar, like "might fail with an error"

Suggestion that you can accept if you want is in the next comment